package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "pytest-html",
+  "version": "1.10.0",
```

Comment:
Is it necessary to create a package? I'd like to at least avoid maintaining the version number in multiple places, so if we need this please change the version to something like 0.0.1 so it's clear we're not needed to keep it in sync with the python package.